app/src/org/commcare/dalvik/activities/CommCareHomeActivity.java

```diff
@@ -716,9 +716,6 @@ else if(resultCode == RESULT_OK) {
                     break;
                 } else if(resultCode == RESULT_OK) {
                     currentState.getSession().setDatum(currentState.getSession().getNeededDatum().getDataId(), intent.getStringExtra(SessionFrame.STATE_DATUM_VAL));
-                    if(intent.hasExtra(CallOutActivity.CALL_DURATION)) {
-                        platform.setCallDuration(intent.getLongExtra(CallOutActivity.CALL_DURATION, 0));
-                    }
```

Comment:
According to @ctsims, all the call duration logic can be removed, so we shouldn't have to worry to hard about taking out something important